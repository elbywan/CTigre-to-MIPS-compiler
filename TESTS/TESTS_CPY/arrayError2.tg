type intArray = array of int in
var tab := intArray[12] of 0 in
printint(tab[12]);
print("dead code\n")