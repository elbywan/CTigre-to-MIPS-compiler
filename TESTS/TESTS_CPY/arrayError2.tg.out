ERROR: array index out of bound

